### PR TITLE
Consitently capitalize first letters of wiki titles

### PR DESCRIPTION
**Pull request by bux**

### CODE REVIEW

File: docs/wiki/development/how-to-translate-ace3.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: wiki
-title: How to translate ACE3
+title: How To Translate ACE3
```
Comment (synixebrett) [inline docs/wiki/development/how-to-translate-ace3.md]: `To` should probably remain `to`

File: docs/wiki/user/ace3-and-acex.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: wiki
-title: ACE3 and ACEX
+title: ACE3 And ACEX
```
Comment (synixebrett) [inline docs/wiki/user/ace3-and-acex.md]: `And` should probably remain `and`

File: docs/wiki/user/how-to-report-an-issue.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: wiki
-title: How to report an issue
+title: How To Report An Issue
```
Comment (synixebrett) [inline docs/wiki/user/how-to-report-an-issue.md]: `to` and `an` should probably remain lowercase

**Comment by bux** [comment]
@synixebrett is that some kind of "rule" in the english language? I wasn't sure if every first letter should be capitalized or not.

**Comment by Drofseh** [comment]
English has `Title Case` where only principal words are capitalized.
Articles (a, an, the, etc.), conjunctions (and, but, or, etc.), prepositions (to, on, in, with, etc.) are not capitalized unless they are the first word in the title.

File: docs/wiki/user/ace3-and-acex.md (hunk 2 above)
Comment (dedmen) [reply docs/wiki/user/ace3-and-acex.md]: Also felt wrong to me but didn't know if it was supposed to be like that

**Comment by commy2** [comment]
The Lord of the Rings

**Pull request merged**